### PR TITLE
Added Queue using linkedlist 

### LINKED ISSUE #3777: Queue implementation using linkedlist
## 💥 Proposal

Want to add implementation of Queue using linkedlist in cpp
Have gone through all the issues created previously, there is no such similar issue created:)

### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#reporting-new-issues)?
Yes

### PR DESCRIPTION
**Pull request by Subhangini**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

Added the operations which can be performed on queue using linkedlist
### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

fixes #3777

### CODE REVIEW

File: C-Plus-Plus/Queue/Queue_all_operations_using_linkedlist/program.cpp [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+        default:
+            cout << "\nWANT TO DO NOTHING! IT's OK!";
+        }
+        cout << "\nWant to insert or delete an another node?(Y/N):";
+        cin >> ch;
+    } while (ch == 'y' || ch == 'Y');
+
+    return 0;
+}
+/* Time complexity for both insertion and deletion is:- O(1)
```
Comment (siddharth25pandey) [inline C-Plus-Plus/Queue/Queue_all_operations_using_linkedlist/program.cpp]: Write Sample Input and Output at the end of the Program in Commented form

File: C-Plus-Plus/Queue/Queue_all_operations_using_linkedlist/program.cpp [hunk 2]
```diff
@@ -0,0 +1,105 @@
+/*To know about what queue is and what is the workflow of this program see the README.md file of this folder*/
+#include<iostream>//Header file for input output operations
```
Comment (siddharth25pandey) [inline C-Plus-Plus/Queue/Queue_all_operations_using_linkedlist/program.cpp]: Use **bits/stdc++**

**Review by siddharth25pandey — CHANGES_REQUESTED** [review]
@Subhangini Do the small changes, rest is fine

File: C-Plus-Plus/Queue/Queue_all_operations_using_linkedlist/program.cpp (hunk 1 above)
Comment (Subhangini) [reply C-Plus-Plus/Queue/Queue_all_operations_using_linkedlist/program.cpp]: ok

**Comment by siddharth25pandey** [comment]
@Subhangini Also, you need to update the pre-existing **README** file. No need to add a new **README file**

File: C-Plus-Plus/Queue/Queue_all_operations_using_linkedlist/program.cpp [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+   Space Complexity:- O(n)*/
+/* Test case
+Sample Input:
+1
+12
+Sample Output:
+New node with value 12 is inserted in Queue.
+*/
+
+
```
Comment (siddharth25pandey) [inline C-Plus-Plus/Queue/Queue_all_operations_using_linkedlist/program.cpp]: Sample I/O is **not matching** with the execution of the Program

**Review by siddharth25pandey — CHANGES_REQUESTED** [review]

**Comment by Subhangini** [comment]
> @Subhangini Also, you need to update the pre-existing **README** file. No need to add a new **README file**

I have added that readme file in order to give a better understanding of the program and data structure

should i remove this readme?

File: C-Plus-Plus/Queue/Queue_all_operations_using_linkedlist/program.cpp (hunk 3 above)
Comment (Subhangini) [reply C-Plus-Plus/Queue/Queue_all_operations_using_linkedlist/program.cpp]: ok i will change this

**Comment by siddharth25pandey** [comment]
> > @Subhangini Also, you need to update the pre-existing **README** file. No need to add a new **README file**
> 
> I have added that readme file in order to give a better understanding of the program and data structure
> 
> should i remove this readme?

@Subhangini Yes, you need to remove it & update the **README** file. For more info, check the merged PR. 
Also, change the path of the Program to the **`cpp/ds/`**. No need to create a Separate folder for **Queue**

**Comment by Subhangini** [comment]
> > > @Subhangini Also, you need to update the pre-existing **README** file. No need to add a new **README file**
> > 
> > 
> > I have added that readme file in order to give a better understanding of the program and data structure
> > should i remove this readme?
> 
> @Subhangini Yes, you need to remove it & update the **README** file. For more info, check the merged PR.
> Also, change the path of the Program to the **`cpp/ds/`**. No need to create a Separate folder for **Queue**

ok

**Comment by siddharth25pandey** [comment]
@Subhangini Resolve the merge conflict

File: C-Plus-Plus/README.md [hunk 4]
```diff
@@ -122,6 +122,7 @@
 
 ## Data Structures
 
+- [Queue all operations using linked list](ds/Queue_all_operations_using_linkedlist.cpp)
```
Comment (ankitaggarwal23) [inline C-Plus-Plus/README.md]: Rewrite it as `Queue using linked list`

**Review by ankitaggarwal23 — CHANGES_REQUESTED** [review]
- Give proper spacing.
- Use proper code format
- Remove redundant comments due to so many comments code looks messy and not readable

**Review by siddharth25pandey — APPROVED** [review]
@Subhangini LGTM 💯 Able to merge

**Comment by siddharth25pandey** [comment]
@all-contributors please add @Subhangini for her code

**Comment by Subhangini** [comment]
Finally,🎉 Thank you for merging 
Glad to contribute in this project

**Pull request merged**